### PR TITLE
Improve wording on find_by note regarding returning only one record [ci skip]

### PR DESCRIPTION
**Pull request by rousisk**
This is mostly about replacing "registry" with "record", but I thought that I could probably rewrite the whole note to be more succinct. Any feedback is more than welcome.

### CODE REVIEW

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -1403,8 +1403,9 @@ WHERE people.name = 'John'
 LIMIT 1
 ```
 
-NOTE: Remember that, if `find_by` returns more than one registry, it will take
-just the first and ignore the others. Note the `LIMIT 1` statement above.
+NOTE: Note that if a query matches multiple records, `find_by` will
+fetch only the first one and ignore the rest (see the `LIMIT 1`
```
Comment (robin850) [inline guides/source/active_record_querying.md]: Why did you use "the rest" instead of "the others" ?

Comment (rousisk) [inline guides/source/active_record_querying.md]: I just thought that it reads better but I might be wrong! Do you think that "the others" is more suitable?

Comment (robin850) [inline guides/source/active_record_querying.md]: That's really a nit-picky thing but yep, I think "the others" is more clear ; it explicitly refers to the ignored records while "the rest" is a bit vague.

Comment (rousisk) [inline guides/source/active_record_querying.md]: Yes, I see your point. I changed it back.

**Comment by robin850** [comment]
Thank you very much!

**Pull request merged**